UnitedIncome/components
# Updating InfoBox to meet styling guides

### Description
> Please provide a brief description of your changes below.

Added a phantom div that will expand to take up the full size of the card on hover with long names. Also updated tests. Also corrected title padding.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

Enter a long title for the InfoBox component and ensure that when you hover over the title a full-size `$midnight` overlay is provided with the full title text of the card.

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

_Before Hover_
<img width="365" alt="Screen Shot 2019-12-03 at 1 46 08 PM" src="https://user-images.githubusercontent.com/8663031/70079822-785f0f80-15d3-11ea-8a90-fb2611adeff6.png">

 _Hover_
<img width="365" alt="Screen Shot 2019-12-03 at 1 46 44 PM" src="https://user-images.githubusercontent.com/8663031/70079830-7e54f080-15d3-11ea-8b5d-1b6793933857.png">

### Comments
> Any additional comments that will aid with the review process.

Tested on dev using `yarn link` to ensure that these changes meet the specifications 👍🏽

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-4064)
